### ISSUE TITLE
[Feature Request]:  Please add an option to disable AI audio dubbing

### ISSUE THREAD

**Issue by wurmtal868** [yuliskov/SmartTube#4220]
### Describe the feature you'd like to request

I'm a native German speaker but also fluent in English.
I watch popular channels from German and English speaking countries.
Lately more and more videos get automatically audio dubbed by AI.
The only way to watch videos in the original version is switching the player's audio language between German and English.
Lately it feels like I have to switch for every video I watch, which is annoying like watching advertisements.
An option to pick the original video language would be nice.

### Describe the solution you'd like

Please add an option to disable AI audio dubbing!

### Describe alternatives you've considered

Enable multiple audio language picks in the player settings and switch to the original language when the video language matches one of the picks.

**Comment by Nadury16** [comment]
Yes please, it’s really annoying. I have it selected to None and it translates to french and I don’t even speak it

**Comment by bionictom** [comment]
Agreed. It is a big annoyance how that changed a couple of weeks ago.

**Comment by lvfmc85** [comment]
I’m experiencing the same issue, and setting the audio language to 'none' produces strange results. I have channels in English, Portuguese, and Spanish, and I prefer to watch them in their original audio. When I select 'none' as the audio language, some of my English channels play German translations, and some Portuguese and Spanish channels play English translations. Ideally, the 'none' option would default to the original audio, but that doesn't happen. It’s annoying to deal with these automatic AI translations and have to manually change the audio every time I watch content in a different language.

**Comment by N3orun** [comment]
+1 for that. This `feature` is so annoying.  Would love to see that option here.

**Comment by sbirulicchio95** [comment]
Same Here. It's really really annoying. We need a global feature to disable it once for all.

**Comment by totoro452** [comment]
And same here.
PLEASE :-)

**Comment by Section25** [comment]
Possibly not related but I didn't want to start a new feature request; I'd love to see a way to completely filter out AI and/or text-to-speech content.

I don't know how feasible this might be but there is an ad-blocker, SponsorBlock and DeArrow so maybe it could be achieved.

**Comment by M4SSK** [comment]
Indeed, this is getting really annoying.
I managed to minimize the problem for some time by choosing "Cherokee" as default audio language. As there was almost never an audio dubbing track for this language, Youtube played the original audio. But there seems to be a new trend of AI dubbing that defaults to English AI dubbing if anything other than the original language is selected !

One solution, if completely disabling AI dubbing is not possible, would be to add the option to remember the audio track selected for each channel (the same way you can already remember the subtitles languages for each channel individually)

**Comment by tm-frs** [comment]
+1 to this, i hate how AI shit is being forced on everyone, i mean yes, cool ootion to have if i really needed to watch a video in a language i dont speak, but it shouldn't be enabled by default or at least it should have an option to turn it off. So I'd be really happy to see such a thing implemented in this app

**Comment by Lichtblitz** [comment]
ReVanced has a patch to disable the AI dubs for the original Android YouTube apps. So it is possible and maybe the patch can be used as a reference to do the same in SmartTube.

This might be the relevant code and it seems it's as easy as selecting the audio track with the name "original"

https://github.com/ReVanced/revanced-patches/pull/4122/files#diff-421c33b1506ff5984d0ee9afd098741d4b1226a3f206be5cdab9d12a40805a25R29

**Comment by martijk** [comment]
This was released recently:

> Improved audio language preference: disable AI audio translation by selecting the Default option (Settings/Player/Audio language/Default); languages list reworked - the recent selection is now on the top of the list

https://github.com/yuliskov/SmartTube/releases/tag/25.24s

**Comment by ruben2000de** [comment]
> What helped me was: settings - player - audio language - none

fixed it, for me. Thank you!

**Comment by guitarnut1** [comment]
This just started in the past week only on one channel that's native Spanish. It automatically overdubs in English and I speak Spanish and it sounds ridiculous. So I always have to go into that video settings and change it. I'll see if there's a global setting for all videos but this bug is new. It never did this before.

**Comment by martijk** [comment]
> This just started in the past week only on one channel that's native Spanish. It automatically overdubs in English. I speak Spanish and don't want English. It also sounds ridiculous. So I always have to go into that video settings and change it. I'll see if there's a global setting for all videos but this bug is new. It never did this before.

Make sure that you're running version 25.24 or later and that you select "Default" under Settings/Player/Audio language. See https://github.com/yuliskov/SmartTube/releases/tag/25.24s.

**Comment by guitarnut1** [comment]
> > This just started in the past week only on one channel that's native Spanish. It automatically overdubs in English. I speak Spanish and don't want English. It also sounds ridiculous. So I always have to go into that video settings and change it. I'll see if there's a global setting for all videos but this bug is new. It never did this before.
> 
> Make sure that you're running version 25.24 or later and that you select "Default" under Settings/Player/Audio language. See https://github.com/yuliskov/SmartTube/releases/tag/25.24s.

The Default setting worked thanks. It was set to English. I am at 25.60 beta.

**Comment by GithubUser99999999** [comment]
The "default" setting is not a viable solution. I want to watch German and English content without having to set anything as default in order to suppress absurd AI translations, because I just want to watch the respective content in whatever original language it comes in.

**Comment by N3orun** [comment]
I have set my global language to english, using default, not getting any german ai dubbing.

**Comment by martijk** [comment]
> The "default" setting is not a viable solution. I want to watch German and English content without having to set anything as default in order to suppress absurd AI translations, because I just want to watch the respective content in whatever original language it comes in.

To be clear: you should select the literal string "default" as the global option in the settings menu rather than choosing one of the listed languages. From then on it picks the default (original) audio stream instead of AI dubs, unless you've saved other preferences for a particular channel.

**Comment by MarvinMynx** [comment]
I'm having the same issue, but I can't even select a non-AI-translated audio track — I only have German AI-dubbed tracks available. Everything else still plays in the original audio, except for one channel I like to watch. I haven’t been able to watch it at all for at least a week — the entire channel is AI-dubbed for me right now.

The default language setting, as many others have described before me, does absolutely nothing. I’ve tried setting it to German, but all my other native English videos still play in English, and that one channel remains stuck in AI German. I tried setting it to "" or "none", as suggested here — same result. I tried English — same result. I tried Default — same result. I even tried "Cherokee," as described by another user above — still the same result.

No matter what setting I choose, I can’t select any other audio track than the AI-dubbed German — and only for that one channel.

I'm currently on Version 27.92 Beta.

Please help 🥺

It's the Game Grumps channel, by the way — and honestly, not watching the videos is better than listening to the most horrible translation I’ve ever heard, in any context, ever...

**Comment by MarvinMynx** [comment]
> I’m experiencing the same issue, and setting the audio language to 'none' produces strange results. I have channels in English, Portuguese, and Spanish, and I prefer to watch them in their original audio. When I select 'none' as the audio language, some of my English channels play German translations, and some Portuguese and Spanish channels play English translations. Ideally, the 'none' option would default to the original audio, but that doesn't happen. It’s annoying to deal with these automatic AI translations and have to manually change the audio every time I watch content in a different language.

I can't even Change to a none ai translated one s d only German...

**Comment by RomeoAlphaEkko** [comment]
german here, happened to me today too, got english translated to german and i only see german 

under audio formats i only see de-de (deactivating means no audio at all)

audio language is set do BLANK, it doesnt change if i go to standard or even english

but now i learned that pokémon is doing legal business for pokimane

**Comment by 50P15** [comment]
Same here, the "settings - player - audio language -  default" didn't work, until I set smarttube language to English.
I set my language back again, and it's still working fine.

Seems it doesn't save the setting properly if the app language isn't in English.

**Comment by nerijus** [comment]
It is now settings -> player -> audio language -> Original.

**Comment by mastacheata** [comment]
> Same here, the "settings - player - audio language - default" didn't work, until I set smarttube language to English. I set my language back again, and it's still working fine.
> 
> Seems it doesn't save the setting properly if the app language isn't in English.

Yeah, can confirm - it's somehow related to the UI language of SmartTube itself. If you set the UI language to German some of the Video titles are translated and some videos use the horrible AI dub feature even if the Audio Language is set to Original

It also seems like the creators aren't aware of that feature and have to explicitly turn it off for every single video they publish.

**Comment by baptistebisson** [comment]
I'm also having this annoying issue. The interface lanugage is used everytime AI audio is available in my language. 

In the player, clicking on HQ and then audio track, there is no way of selecting another language track.

**Comment by xorinzor** [comment]
Since the last few days even setting the audio to "original" it still plays the english dub on a dutch video.

Can not find any way to get it to play the original audio track.

Setting the audio format to disabled just disables the entire audio track and leaves me with video-only.

Even when I set the audio language to dutch, the only audio formats that show are all English AI dubs. It is straight up unusable currently.

EDIT: the only thing that seems to have worked is to set the app interface language to dutch. Now both dutch and English videos use the original audio.

EDIT 2: there are still random videos that are now AI dubbed to dutch, now that the app language is set to dutch. Oddly doesn't happen for all english videos. Setting audio language to original does not fix it.

**Comment by martijk** [comment]
For me this hasn’t happened ever since the “original” (previously “default”) audio setting was introduced. There must be another setting or usage pattern that makes the difference. My app is set to English and I watch a lot of Spanish content, which always plays fine without any dubs. The same goes for other languages.

**Comment by mastacheata** [comment]
It's the randomness that makes this hard to tell - Creators can turn off the feature when uploading their videos or it might not be available everywhere (depending on the creators location or just random a/b testing of new features on youtube)

**Comment by martijk** [comment]
I’ve just changed the audio language to English and I immediately got the English AI dubs. I changed it back to original and it changes back to Spanish for the same video. The only setting I touched was Player -> Audio language in the settings menu.

**Comment by gsl2018** [comment]
The app was set to my native language. I've changed the language of the app to English and it worked, now I've got the original audio back.

**Comment by tasugo** [comment]
Can confirm this is still happening. I have the app set to English, the audio track configured to original (or English, doesn't matter), and I still hear the audio in Spanish (I'm in Spain). Selecting any audio track (even German, or Hindi, whatever you prefer) keeps playing back the dubbed Spanish track.

**Comment by Akronae** [comment]
Omg this is so annoying I'm a french native speaker and all french videos are playing with dumb AI dubbed audio, I can only select dubbed audio format or none which is no sound. Seriously this is a real pain I don't even know who likes to listen to AI dubbing anyway

**Comment by martijk** [comment]
People who have this issue, please share more details, such as:
- which device you're running it on
- the version of the application
- have you made sure to configure the audio language to "original"
- what language do you use for the interface
- does the issue persist if you change the interface language to English 
- an example of a video that shows this problem 

Since it's not broken for everyone, we need to find out what's breaking it for some.

**Comment by Akronae** [comment]
@martijk for my case the issue disappeared out of the blue, without any update one day after my post (aug 14). Then I had a bug where subscription videos would not show, and doing an update fixed all the issues. So far so good for me 😄

**Comment by mastacheata** [comment]
@martijk

> People who have this issue, please share more details, such as:
> 
>     * which device you're running it on

A Sony TV running Android TV  9

> 
>     * the version of the application

29,23

> 
>     * have you made sure to configure the audio language to "original"

Choosing Original or any other language in the quality screen doesn't change anything. It's always the German AI dub.
Choosing the Original or any other language in the Player Settings from the Settings screen makes no difference either.

>     * what language do you use for the interface

German (Deutsch)

 
>     * does the issue persist if you change the interface language to English

Not in this case. I switched the interface language to English - no AI dub then back to German and the AI dub is gone as well. I had restarted the App 2-3 times before in order to grab all the version info and that didn't change anything, maybe it's if there's already some info about the video in the App's Cache?

>     * an example of a video that shows this problem

https://youtu.be/6mmPkqVzhb4?si=yhk89R5n9E2Ptxhi


It's really random, only some videos behave that way and usually if you try to reproduce it next week with a new update it's gone again and then returns. 
This is by far not most videos I watch, but there are broken AI dub videos all the time.


HTH

**Comment by mastacheata** [comment]
> [@martijk](https://github.com/martijk) for my case the issue disappeared out of the blue, without any update one day after my post (aug 14). Then I had a bug where subscription videos would not show, and doing an update fixed all the issues. So far so good for me 😄


That's a common theme here as well. The same video will have no problems if you leave it untouched and just wait for the next SmartTube update before you try again.

**Comment by marekorok** [comment]
I got similar issue and come to this thread.
version 29.37
I'm on the Polish interface and on my audio formats I had only pl(dubbed-auto) available and nothing else

Switching interface to English and then back to Polish solved problem in my case, now I have back all possible audio formats including other-dubbed languages and English(original) as well.

problem solved, thanks for tips

**Comment by RM6SS62** [comment]
Is there a valid reason because there is no the chance to switch between available audio languages in the video itself, just like in youtube?
